src/internal/transactionenv/txncontext/context.go

```diff
@@ -50,11 +47,10 @@ func New(ctx context.Context, sqlTx *sqlx.Tx, authServer identifier, m FilesetMa
 		}
 	}
 	return &TransactionContext{
-		SqlTx:          sqlTx,
-		CommitSetID:    uuid.NewWithoutDashes(),
-		Timestamp:      types.TimestampNow(),
-		username:       username,
-		FilesetManager: m,
+		SqlTx:       sqlTx,
+		CommitSetID: uuid.NewWithoutDashes(),
+		Timestamp:   types.TimestampNow(),
```

Comment:
We have the transaction, can we do `select CURRENT_TIMESTAMP` ?

Reply:
Ah, didn't realize SQL transactions had an associated timestamp, that makes sense to me

Reply:
It doesn't do anything fancy with transaction ordering/retrying.  It's just from the start of the transaction.  But I figure we should try to use the database clock where we can.